public/stylesheets/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  margin: 0;
+  min-width: 960px;
+  border-top: 10px solid #b71f39;
+
+}
+
+.grapes {
```

Comment:
Is this your intention with the grapes image? Just checking since I think that the priority for you is the nav-bar.